### DISCUSSION TITLE
Adding images to these blocks

### DISCUSSION THREAD

**Discussion by ajeetraina** [facebook/docusaurus#4483] [Q&A]
I am working on building howtos for https://developer.redislabs.com/howtos/ page. Any idea how can I add images to these columns. A snippet of code would be useful.

```
<div class="row">

 <div class="col">
     <RedisCard 
        title="Rate Limiting" 
        description="Implementing Rate Limting" 
        preview="ratelimiting.png"
        page="/howtos/ratelimiting/"
        />
</div>


 <div class="col">
    <RedisCard 
        title="Shopping Cart" 
        description="Implementing Shopping Cart" 
        page="/howtos/shoppingcart/"
        />
 </div>
</div>
```

**Comment by slorber** [comment]
Like `<img src="/img/myImage.png"/>` and having a file in `static/img/myImage.png`

 or `<img src={require("./myImage.png").default}/>` and having the image in the same folder as the page component?
